### ISSUE TITLE
Legacy keys vs Grid keys, small inconsistency, missing keys

### ISSUE THREAD

**Issue by nullaegy** [beyond-all-reason/Beyond-All-Reason#3492]
### Description

I've started with legacy keys and I wanted to switch to grid hotkeys, but there are missing features if I use one or another.

In legacy mode, missing some keys: 
- there is no camera save (CTRL F1-F4)
- no split army like in grid keys (CTRL+Q)
- there is no resurrect key (grid CTRL+W)

The problem is gird keys are lack some features as well:
- track unit (legacy T key) (this is what I miss the most)
    maybe ALT+t would be a good alternative, or maybe ALT+t for toggle, so T can be used to track and ALT+t could be the toggle.
- can't remove from auto-group. Probably because of the [Hungarian keyboard layout](https://commons.wikimedia.org/wiki/File:KB_Hungary.svg).
    Maybe this could be done with a dedicated UI button, or something not so special key, like ALT+G
- can't use fight or attack command from any lab if there is a unit in the F place. If I hover over the fight button in the UI, it says F.
    for F maybe Ctrl+F would be a solution.
- not so big deal, but sometimes for example radar and naval radar hotkeys (and build buttons as well) aren't necessary, because its placement will decide what would you like to place, but it steals a place and a hotkey.
- In game top right "Keys" menu, both in legacy and grid keys has the description "FUN" for ALT+F4, which is fun for the old dogs like myself, but not for new players who press it in an online lobby. It should be "Abandon game / force quit game." or something similar.

Keep up the good work, I love this game <3

### Expected Behaviour

Same functions available in both mode.

### Actual Behaviour

Slight differences.

### Reproduction steps

Start a game and try to use these keys in both mode.

**Comment by Hazy-uhyR** [comment]
> The problem is gird keys are lack some features as well:
> 
>     * track unit (legacy T key) (this is what I miss the most)

What do you use it for?

>     * can't remove from auto-group. Probably because of the [Hungarian keyboard layout](https://commons.wikimedia.org/wiki/File:KB_Hungary.svg).
>       Maybe this could be done with a dedicated UI button, or something not so special key, like ALT+G

Try Alt-§ to remove from autogroup? Should be the same bind for both presets afaik.

**Comment by nullaegy** [comment]
> track

I use the track units for a lot of stuff, for example it looks so cool :] 
but other than that the bombing run is much better, and if I find some unit which I want to send to battle, and I hit Ctrl+W to select all the same type, and I can focus on them with track, without adding them to group.

And also it's handy if I just want to find where are my idle builder, and I know I can click on the lower bar with right click, but with track it's easier for me to keep the focus of all my selected units, and that way I don't have to deal with the camera movement.

Or if I know I have some unit in the battlefield, which I want to use, so I press CTRL+E to select everyone, than find the tiny avatar :] select only those, and if I want to find them I have to add them to a group, so I can right-click on them, or press twice the corresponding group number. 

I like more the grid keys, so I continue to use it, instead of legacy keys, and I can live with it's functions, most of them are way better, but the track function is still missing :)

> autogroup

Alt-§, doesn't work (tried with AltGr or Alt). Does the same: add the selected units to autogroup 0.
I've tried with all the mod keys(Ctrl, Shift, Alt, AltGr and the top row 0-9 and the accents(öüó) as well. None of them removes them from autogroup. I can live without this feature, but still a nice hotkey that I miss from booth legacy and grid keys as well if I use my hungarian layout.

> radar

About the radar, I'm not sure, because you can't place a radar on water, and can't place a radar/sonar on land maybe there is no need to have them booth on hotkey nor on interface, but it's up for debate, and not about hotkeys as well, maybe a different issue.

**Comment by Hazy-uhyR** [comment]
> > autogroup
> 
> Alt-§, doesn't work (tried with AltGr or Alt). Does the same: add the selected units to autogroup 0. I've tried with all the mod keys(Ctrl, Shift, Alt, AltGr and the top row 0-9 and the accents(öüó) as well. None of them removes them from autogroup. I can live without this feature, but still a nice hotkey that I miss from booth legacy and grid keys as well if I use my hungarian layout.

Oh, I didn't even see the 0 there. "Key left of 1" is usually also the ping button, so you cannot ping either.

**Comment by nullaegy** [comment]
yep, ping is a bit tricky, if I hold down the 0 there, the ping and draw works, but not as smooth, sometimes only the second ping works, but draw works ok.

**Comment by abulka** [comment]
Isn't Track a fundamental feature (I LOVE it too) - why has it been removed? Viz. Select a unit, track - then the camera follows the unit. Great for following a unit going into battle or a bomber run.

I've been on a wild goosechase trying to find out why T track is not working, then discovered the legacy vs. grid key mappings is the problem. The new default (grid) key mappings simply don't have Track - why would it be left out? Am I missing something?

**Comment by McMarius11** [comment]
it would be a simple fix in my book,
they should change Ctrl+sc_t  toggleoverview to Ctrl+sc_**m**
and add Ctrl+sc_**t**  track

**Comment by Nick-Deny** [comment]
Was there any update on this? Looking for tracking in grid menus but can't seem to find one - would be super neat to add this back for spectators as well!

**Comment by TheChronographer** [comment]
> Was there any update on this? Looking for tracking in grid menus but can't seem to find one - would be super neat to add this back for spectators as well!

You can always add it back with custom keys.

As for rearranging the existing grid keys to accommodate the differences, this is very unlikely. Removing keys people are used to (like overview) is a very high hurdle to clear.

As for adding the in modifier layers, like alt+t and alf+f, this is much less of a barrier but to be honest there hasn't been much of a demand for things like track and trackmode. They are very rarely used by most players. And those few that want it can add it via custom keys.

The fight from lab issue, might be worth resolving. But it's not as simple as also having alt+f bound to fight as that adds the f grid unit to the front of the factory build queue. It would have to be somewhere away from the grid entirely. However you can always click the order menu gui, which shows fight (unless you've hidden it) and you're not generally issuing many such orders to labs as to need it quickly accessible.

**Comment by mtlam** [comment]
In the [tooltips](https://github.com/beyond-all-reason/Beyond-All-Reason/blob/cbf86c84e991559b2cc6d7ec2408d51bc93e4d3d/language/en/tips.json#L40), it says to press F4 to enable resource view. If you default to grid keys rather than the legacy layout though, you start with it being F7. I think it is better for a new-ish player like me who does default to grid keys to see the tip as it being F7.
